lib/db/mysql/schema.sql

```diff
@@ -41,7 +41,7 @@ CREATE TABLE IF NOT EXISTS tokens (
   FOREIGN KEY (clientId) REFERENCES clients(id) ON DELETE CASCADE,
   userId BINARY(16) NOT NULL,
   INDEX tokens_user_id(userId),
-  email VARCHAR(256) NOT NULL,
+  email VARCHAR(256),
```

Comment:
Yea, that'd be ideal. However, I don't know my way around the auth server much, and to completely drop this column, the auth server would need that PR merged, or else **ALL** requests for `email` would no longer be serviceable.

I didn't want to block landing this based on that, but if it's possible to get that landed asap, then we could just drop all the `email` columns in the oauth db.